Packs/HatchingTriage/ReleaseNotes/1_0_11.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate URL - Hatching Triage
+
+- %%UPDATE_RN%%
```

Comment:
Please update the release note, see https://xsoar.pan.dev/docs/documentation/release-notes.